.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  workflow_run:
+    workflows: ["Validate & Build"]
+    branches:
+      - main
+    types:
+      - completed
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build & Push Container
+      uses: docker/build-push-action@v2
+      with:
+        push: true
+        tags:
+        - srobo/discord-gated-entry:build-${{ github.run_id }}
```

Comment:
Question: What's the thought behind this additional tag?

Reply:
Mostly so that we don't need to manually use git tags to trigger builds. I think we do want some form of tracking for which revisions have been built/published (maybe a better solution is to have github actions create tags or something), rather than just a perineal "latest".

Is there any disadvantage to having these?

Reply:
Not really. Although I suspect what would be more useful to us is to tag based on the commit SHA rather than the run id so it's easier to track through.